test/powershell/Modules/Microsoft.PowerShell.Core/RemoteImportModule.Tests.ps1

```diff
@@ -24,6 +24,8 @@ Describe "Remote import-module tests" -Tags 'Feature','RequireAdminOnWindows' {
         if ($IsWindows) {
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-windows`
Dot-source operator is not allowed in modules on locked down systems that export functions with wildcards.Error occurred in Describe block
```powershell
Cannot bind argument to parameter 'Name' because it is null.
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\ConstrainedLanguageModules.Tests.ps1:251 char:27
```
Import trusted manifest files in system lockdown mode.Verifies that trusted manifest exports no functions through the cmdlets export keyword
```powershell
Cannot bind parameter 'ErrorAction'. Cannot convert value "ImportTrustedManifestWithCmdletExport_System32" to type "System.Management.Automation.ActionPreference". Error: "Unable to match the identifier name ImportTrustedManifestWithCmdletExport_System32 to a valid enumerator name. Specify one of the following enumerator names and try again:
SilentlyContinue, Stop, Continue, Inquire, Ignore, Suspend, Break"
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\ConstrainedLanguageModules.Tests.ps1: line 1078
1078:                 Remove-Module ImportTrustedManifestWithCmdletExport_System32 -Force -ErrorAction ImportTrustedManifestWithCmdletExport_System32

```
New-Module should not create module from trusted scriptblock when running in ConstrainedLanguage context.New-Module succeeds in creating module with untrusted scriptblock in ConstrainedLanguage
```powershell
Cannot bind parameter 'ModuleInfo'. Cannot convert the "UntrustedScriptFoo" value of type "System.String" to type "System.Management.Automation.PSModuleInfo".
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\ConstrainedLanguageModules.Tests.ps1: line 1557
1557:                 Remove-Module -ModuleInfo UntrustedScriptFoo -Force -ErrorAction SilentlyContinue

```